Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MainActivity.cs

```diff
@@ -1,55 +1,80 @@
-﻿using System;
-
-using Android.App;
-using Android.Content;
+﻿using Android.App;
+using Android.OS;
+using Android.Support.V7.App;
 using Android.Runtime;
-using Android.Views;
 using Android.Widget;
-using Android.OS;
 
 using Android.Util;
-using Gcm.Client;
-
+using Android.Gms.Common;
 
 namespace GetStartedXamarinAndroid
 {
-	[Activity (Label = "GetStartedXamarinAndroid", MainLauncher = true, Icon = "@drawable/icon")]
-	public class MainActivity : Activity
-	{
-		int count = 1;
-		public static MainActivity instance;
-
-		protected override void OnCreate (Bundle bundle)
-		{
-			instance = this;
-
-			base.OnCreate (bundle);
-
-			// Set our view from the "main" layout resource
-			SetContentView (Resource.Layout.Main);
-
-			// Get our button from the layout resource,
-			// and attach an event to it
-			Button button = FindViewById<Button> (Resource.Id.myButton);
-			
-			button.Click += delegate {
-				button.Text = string.Format ("{0} clicks!", count++);
-			};
-
-			RegisterWithGCM();
-		}
-
-		private void RegisterWithGCM()
-		{
-			// Check to ensure everything's set up right
-			GcmClient.CheckDevice(this);
-			GcmClient.CheckManifest(this);
-
-			// Register for push notifications
-			Log.Info("MainActivity", "Registering...");
-			GcmClient.Register(this, Constants.SenderID);
-		}
-	}
-}
+    [Activity(Label = "@string/app_name", Theme = "@style/AppTheme", MainLauncher = true)]
+    public class MainActivity : AppCompatActivity
+    {
+        public const string TAG = "MainActivity";
+        internal static readonly string CHANNEL_ID = "my_notification_channel";
```

Comment:
The `CHANNEL_ID` should be renamed to `ChannelId` to conform to .NET Naming standards
Why is this internal static readonly instead of a constant?  
Should this be in the constants file because we make no mention of it in our docs down below?

Reply:
Yes. Making it a constant.